From 7d4f618ac16563e914cda9753c4a54b5d5059d89 Mon Sep 17 00:00:00 2001
From: "Juan A. Suarez Romero" <jasuarez@igalia.com>
Date: Sun, 11 Dec 2016 16:10:41 +0100
Subject: [PATCH] core: include @filename@ in file-production

Small fix for an include that is in the wrong place.

https://bugzilla.gnome.org/show_bug.cgi?id=775950
---
 src/grl-type-builtins.c.template | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/grl-type-builtins.c.template b/src/grl-type-builtins.c.template
index 734d23a..1189a4c 100644
--- a/src/grl-type-builtins.c.template
+++ b/src/grl-type-builtins.c.template
@@ -1,10 +1,10 @@
 /*** BEGIN file-header ***/
-#include "grl-type-builtins.h"
-#include "@filename@"
+#include <grl-type-builtins.h>
 /*** END file-header ***/
 
 /*** BEGIN file-production ***/
 /* enumerations from "@filename@" */
+#include "@filename@"
 /*** END file-production ***/
 
 /*** BEGIN value-header ***/
-- 
1.8.3-rc3

